packages/components/src/components/hds/icon-tile/types.ts

```diff
@@ -0,0 +1,52 @@
+export enum HdsIconTileSizeValues {
+  Small = 'small',
+  Medium = 'medium',
+  Large = 'large',
+}
+
+export type HdsIconTileSizes = `${HdsIconTileSizeValues}`;
+
+export enum HdsIconTileProductValues {
+  Boundary = 'boundary',
+  Consul = 'consul',
+  HCP = 'hcp',
+  Nomad = 'nomad',
+  Packer = 'packer',
+  Terraform = 'terraform',
+  Vagrant = 'vagrant',
+  Vault = 'vault',
+  'Vault Secrets' = 'vault-secrets',
+  'Vault Radar' = 'vault-radar',
+  Waypoint = 'waypoint',
+}
+
+export type HdsIconTileProducts = `${HdsIconTileProductValues}`;
+
+// TODO - figure out how to avoid duplicating most of the enum values
+export enum HdsIconTileColorValues {
+  Neutral = 'neutral',
+  Boundary = 'boundary',
+  Consul = 'consul',
+  HCP = 'hcp',
+  Nomad = 'nomad',
+  Packer = 'packer',
+  Terraform = 'terraform',
+  Vagrant = 'vagrant',
+  Vault = 'vault',
+  'Vault Secrets' = 'vault-secrets',
+  'Vault Radar' = 'vault-radar',
+  Waypoint = 'waypoint',
+}
```

Comment:
We could use a union type to avoid duplication here
```
enum HdsIconTileProductValues {
  ...
}

enum HdsIconTileNeutral {
  Neutral = 'neutral',
}

type HdsIconTileColorValues = HdsIconTileProductValues | HdsIconTileNeutral; 
```

Reply:
Nice! That did the trick.